src/pages/groupComparison/GroupComparisonStore.ts

```diff
@@ -397,6 +399,22 @@ export default class GroupComparisonStore {
         invoke:()=>Promise.resolve(this.activeGroups.result![1])
     });
 
+    readonly referenceGenes = remoteData<ReferenceGenomeGene[]>({
+        await: ()=>[
+            this.studies,
+            this.mutationEnrichmentData,
+            this.copyNumberData
+        ],
+        invoke: async () => {
+            const mutGenes = this.mutationEnrichmentData.result!.map(
+                (a:AlterationEnrichment)=>a.hugoGeneSymbol.toUpperCase());
+            const cnvGenes = this.copyNumberData.result!.map(
+                (a:AlterationEnrichment)=>a.hugoGeneSymbol.toUpperCase());
+            return fetchReferenceGenomeGenes(this.studies.result[0].referenceGenome,
+                mutGenes.concat(cnvGenes));
```

Comment:
seems like you have to do the same thing here, right? or else the cytoband column will be empty in group comparison

Reply:
nvm: see my comment https://github.com/cBioPortal/cbioportal-frontend/pull/2180/files#r310723393